corehq/util/log.py

```diff
@@ -96,7 +96,7 @@ def emit(self, record):
         context = self.get_context(record)
 
         message = "\n\n".join(filter(None, [
-            context['message'],
+            self._clean_subject(context['message']),
```

Comment:
I think you're cleaning the wrong thing here - this is the message, which goes in the body of the email.  On line 105 here, `mail.mail_admins` is passed `context['subject']` as the first arg - I believe that's the problem area.